README.md

```diff
@@ -34,9 +34,13 @@ A complete table of available flags:
 
 | Flag    | Description                                           | Example                                                    |
 | --------| ------------------------------------------------------| ---------------------------------------------------------- |
-| inresp  | Read in OCSP response(s) files                        | `./ocsp_status -inresp resps/google_resp` |
-| incert  | Read in certificate files (must be ASN.1 DER encoded) | `./ocsp_status -incert certs/google_cert.der` |
-| ocspurl | Specify the url to send the OCSP request to           | `./ocsp_status -ocspurl=http://ocsp.pki.goog/gts1o1core google.com:443` |
-| post    | Use POST to send the OCSP request (instead of GET)    | `./ocsp_status -post google.com:443` |
-| dir     | Write the OCSP response to a file                     | `./ocsp_status -dir=resps/google_resp google.com:443`|
+| inresp  | Read in OCSP response(s) files                        | `./ocsp_status -inresp google_resp google_resp2` |
+| incert  | Read in certificate files (must be ASN.1 DER encoded) | `./ocsp_status -incert google_cert.der google_cert2.der` |
+| issuercert | Read in space separated issuer certificate files (must be ASN.1 DER encoded) | `./ocsp_status -issuercert="googleissuer_cert.der googleissuer_cert2.der" -incert google_cert.der google_cert2.der` |
+| ocspurl | Read in space separated urls to send the OCSP request to           | `./ocsp_status -ocspurl=http://ocsp1.com http://ocsp2.com google.com:443 google2.com:443` |
+| post    | Use POST to send the OCSP request (default is GET)    | `./ocsp_status -post google.com:443` |
```

Comment:
should this flag be something like `usePost` or `use_post` or `should_use_post` or something to let them know it is a boolean?

Reply:
Done!